src/ui/screens/getHelp/developerMenu/ViewDeveloperMenu.qml

```diff
@@ -290,6 +290,15 @@ MZViewBase {
             }
         }
 
+        MZButton {
+            id: resetTipsAndTricksModal
+
+            text: "Reset Intro Tips and Tricks Modal"
+            onClicked: {
+                MZSettings.tipsAndTricksIntroShown = false
```

Comment:
This modal was removed (about a half hour ago) in #7930 so we can revert this change and the one in ViewHome.qml.